frontend/src/components/LandingPage/Weather/Weather.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './Weather.css'
+
+const weatherData = {
+    city: "Allen" + " ",
+    temperature: "65" + '\xB0F ',
+    picture: "/partly cloudy.png"
+};
+
+class Weather extends React.Component {
+    render() {
+        return(
+            <span>
+               <b>{weatherData.city}</b>
+               {weatherData.temperature}
+               <img className= "medium-icon" src = {weatherData.picture} />
```

Comment:
Remove spaces around the equal signs in the `className` and `src` attributes in `img` tag. It is easier to read when there are no spaces around equal signs in html.

[HTML style guide](https://www.w3schools.com/html/html5_syntax.asp)